server/src/test/java/org/opensearch/search/pipeline/SearchPipelineServiceTests.java

```diff
@@ -215,10 +215,13 @@ private static abstract class FakeProcessor implements Processor {
         private final String tag;
         private final String description;
 
-        protected FakeProcessor(String type, String tag, String description) {
```

Comment:
I think you can remove this `FakeProcessor` base class now, since it's essentially just a copy of `AbstractProcessor`. With `AbstractProcessor` moved under the `server` tree, you can now use it instead.

Reply:
remove FakeProcessor